src/Administration/Resources/app/administration/src/app/component/structure/sw-admin-menu/index.js

```diff
@@ -9,7 +9,7 @@ const { dom, types } = Shopware.Utils;
  *
```

Comment:
this is what the script above is doing as well

the syntax
`Shopware.Component.register('sw-block',` into `export default {`

Shopware.Component.register is a sideeffect in the file and actual file needs to be read in order to be used, not only "the knowledge about the file"